### PR TITLE
IBX-6918: Fix form attribute instructions

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | [IBX-6918](https://issues.ibexa.co/browse/IBX-6918) |
| Versions      | all |
| Edition       | Experience, Commerce |

Fix the tutorial as requested by the customer.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Code samples are working

### CODE REVIEW

**Review by papcio122 — APPROVED** [review]

File: docs/content_management/forms/create_form_attribute.md [hunk 1]
```diff
@@ -53,6 +53,14 @@ The templates for the forms should look as follows:
 [[= include_file('code_samples/forms/custom_form_attribute/templates/themes/standard/formtheme/formbuilder_checkbox_with_richtext_description.html.twig') =]]
 ```
 
+Then, specify the new template in configuration, under the `twig.form_themes` configuration key:
+
+``` yaml
+twig:
+    form_themes:
+        - 'themes/<your-theme>/formtheme/formbuilder_checkbox_with_richtext_description.html.twig'
```
Comment (adamwojs) [inline docs/content_management/forms/create_form_attribute.md:61]: IMHO `@ibexadesign` should be used here

Comment (dabrt) [inline docs/content_management/forms/create_form_attribute.md:61]:
```suggestion
        - '@ibexadesign/formtheme/formbuilder_checkbox_with_richtext_description.html.twig'
```

**Review by mateuszdebinski — APPROVED** [review]

**Review by dew326 — APPROVED** [review]

File: code_samples/forms/custom_form_attribute/public/js/formbuilder-richtext-checkbox.js [hunk 2]
```diff
@@ -13,8 +13,16 @@
...
-})(window, window.document, window.ibexa);
+
+    const openUdw = (config) => {
+        const openUdwEvent = new CustomEvent('ibexa-open-udw', { detail: config });
+
+        doc.body.dispatchEvent(openUdwEvent);
+    };
+
+    ibexa.addConfig('richText.alloyEditor.callbacks.selectContent', openUdw);
+})(window, window.document, window.ibexa);
```
Comment (GrabowskiM) [inline code_samples/forms/custom_form_attribute/public/js/formbuilder-richtext-checkbox.js:28]:
```suggestion
})(window, window.document, window.ibexa);

```

**Review by GrabowskiM — APPROVED** [review]

**Review by julitafalcondusza — APPROVED** [review]

File: docs/content_management/forms/create_form_attribute.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ The new Form attribute requires a `FieldAttributeTypeMapper`. Register the mappe
 
 ## Add Symfony form type
 
-The attribute needs to be editable for the form creator, so it needs to have a Symfony form type. 
+The attribute needs to be editable for the form creator, so it needs to have a Symfony form type.
```
Comment (dabrt) [inline docs/content_management/forms/create_form_attribute.md:33]:
```suggestion
The attribute must be editable for the form creator, so it needs to have a Symfony form type.
```

**Review by juskora — APPROVED** [review]

**Pull request merged**